src/management-system-v2/components/process-list.tsx

```diff
@@ -117,9 +124,33 @@ const ProcessList: FC<ProcessListProps> = ({
             }}
           />
         </Tooltip>
-        <Tooltip placement="top" title={'Delete'}>
-          <DeleteOutlined />
-        </Tooltip>
+        <ProcessEditButton
+          definitionId={record.definitionId}
+          wrapperElement={
+            <Tooltip placement="top" title={'Edit'}>
+              <EditOutlined />
+            </Tooltip>
+          }
+          onEdited={() => {
+            invalidateProcesses();
+          }}
+        />
+        <AuthCan resource="Process" action="delete">
```

Comment:
Since we're dealing with a specific process. the permission check should look like this: `<AuthCan resourse={toCaslResource('Process', object)} action ...`